app/lib/project/ImportController.js

```diff
@@ -75,8 +76,11 @@ define([
                     return true;
                 }.bind( null, this._master )
             };
-            this._sourceGlyphSet = this._project.getGlyphSet(
```

Comment:
Yeah, I agree, it should be refactored. This means `io` needs to be passed down to `_readUFOFormatVersion` and to `readPlist` as well.

It could all be added as an optional last argument to all three methods, then the interface won't change for calling code of these methods.